src/views/create-dataset/CreateDataset.tsx

```diff
@@ -0,0 +1,119 @@
+  const [submitStatus, setSubmitStatus] = useState(false)
+
+  useEffect(() => {
+    setIsLoading(isLoading)
+  }, [isLoading])
+
+  const [state, setState] = useState({
+    createDatasetTitle: ''
+  })
+
+  const handleCreateDatasetFieldChange = (e: ChangeEvent<FormInputElement>) => {
+    let value: (typeof state)[keyof typeof state] = e.target.value
+    if (e.target.type === 'text') {
+      value = e.target.value
+    }
+
+    setState({ ...state, [e.target.id]: value })
+  }
+
+  // TODO: Connect to API when ready
+  const handleCreateDatasetSubmit = (event: FormEvent<HTMLFormElement>) => {
+    setSubmitting(true)
+    event.preventDefault()
+    setTimeout(() => {
+      setSubmitting(false)
+      console.log('Form Submitted!')
+      setSubmitStatus(true)
+    }, 3000)
+    console.log(state)
+    // TODO: Callback for form submission should be here.
+  }
+
+  // TODO: Probably replace this with a FormSkeleton or remove entirely
+  if (isLoading) {
+    return <DatasetSkeleton />
+  }
+  return (
+    <article>
+      <header className={styles.header}>
+        <h1>{t('pageTitle')}</h1>
+      </header>
+      <SeparationLine />
+      <div className={styles.container}>
+        <RequiredFieldText />
+        {submitting && <div>Submtting Form...</div>}
+        <Form onSubmit={handleCreateDatasetSubmit}>
+          {submitStatus && <div>Form Submitted!</div>}
+          <Row>
+            <Col md={9}>
+              <Form.Group controlId="createDatasetTitle" required>
+                <Form.Group.Label>{t('datasetForm.title')}</Form.Group.Label>
+                <Form.Group.Input
+                  data-cy="datasetFormInputTitle"
+                  type="text"
+                  placeholder="Dataset Title"
+                  onChange={handleCreateDatasetFieldChange}
+                />
+              </Form.Group>
+            </Col>
+          </Row>
+          <SeparationLine />
+          <Alert
+            key={1}
+            variant={'info'}
+            customHeading={t('metadataTip.title')}
+            dismissible={false}>
+            {t('metadataTip.content')}
+          </Alert>
+          {/* SRC: https://github.com/IQSS/dataverse/blob/develop/src/main/webapp/dataset.xhtml */}
+          {/* <p:commandButton id="saveBottom" styleClass="btn btn-default" value="#{DatasetPage.editMode == 'CREATE' ? bundle['file.addBtn'] : bundle.saveChanges}" rendered="#{DatasetPage.editMode == 'CREATE' or DatasetPage.editMode == 'METADATA'}"
+              onclick="PF('blockDatasetForm').show();"
+              update=":datasetForm,:messagePanel"
+              oncomplete="$(document).scrollTop(0);"
+              action="#{DatasetPage.save}"
+              /> */}
+          {/* Where is datasetSaveCommand()  */}
+          {/* <p:remoteCommand
+            name="datasetSaveCommand"
+            action="#{DatasetPage.save()}"
+            update=":datasetForm,:messagePanel"
+          /> */}
```

Comment:
I don't understand why this JSF code is commented here. If there's work that needs to be done related to this commented code, we can create an issue and paste this commented code in the issue description. Then, we can add a TODO linking that issue here

Reply:
Better idea. Mostly for immediate recall while I was working on the overall component. I agree it should be moved elsewhere and will keep the snipped above separate for the next issue.